uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-task_test.js

```diff
@@ -0,0 +1,47 @@
+define([
+    'common/angular',
+    'common/moment',
+    'common/angularMocks',
+    'tasks-assignments/app'
+], function (angular, moment) {
+    'use strict';
+
+    describe('ModalTaskCtrl', function () {
+        var ctrl, modalInstance, scope, HR_settings;
+
+        beforeEach(module('civitasks.appDashboard'));
+        beforeEach(inject(function ($controller, $rootScope) {
+            modalInstance = {
```

Comment:
Separate this into its own function, so you can have a more concise `beforeEach` implementation:
```js
ctrl = $controller('ModalTaskCtrl', {
    $scope: scope,
    $modalInstance: fakeModalInstance(), // <---
    data: {},
    HR_settings: HR_settings
 });
```